lib/rapidfire.rb

```diff
@@ -1,4 +1,5 @@
 require "rapidfire/engine"
+require "kaminari"
```

Comment:
Can you try this PR with and without kaminari in a rails app?

Reply:
Yeah, it works out.